workshops/music_app_javascript/README.md

```diff
@@ -0,0 +1,315 @@
+---
+name: 'Music App with JavaScript'
+description: 'Create a cool Music App with Vanilla JavaScript with all your favorite songs.'
+author: '@shibam17'
+img: https://cloud-5v67fzk5g.vercel.app/0thumb.png
+---
+
+Ever wondered to build something which would be fun and for your own along with learning the domain, all of it at once? Well if yes, then you can create your own **Music App** just with JavaScript. Which will not only be fun and creative but also will help to understand term and function in detailed practical manner! :smile: :yum:
+<p align='center'>
+<img src="https://media.giphy.com/media/LmNwrBhejkK9EFP504/giphy.gif" alt="yes-ready-to-code" >
+</p>
+
+## Overview
+In this workshop you will be learning different types of events, functions, methods, classes, styling and more with Vanilla JavaScript, HTML, and CSS.
+This whole workshop is customizable according to you, after all everyone has their own playlist they love to hear in loop.
+The best part is, it will take just around 20 mins to complete this! :relieved: 
+
+*To have a peak in the project:*
+* Final code can be found on [Repl.it](https://repl.it/@shibamdhar/MusicApp) as well as [GitHub](https://github.com/shibam17/musicappjs-workshop).
+* The demo of my music app will be [here](https://musicapp.shibamdhar.repl.co/).
+
+**Prerequisite**
+	Little bit of knowledge on event and event handling from JavaScript can make your work easier but the best part is, even if you are new to this, you can still make it a go without any problem.  :sparkles:
+<p align='center'>
+<img width="300" alt="excited" src="https://media.giphy.com/media/3NtY188QaxDdC/giphy.gif">
+</p>
+
+## Create files and folders
+Since it is a web-app, we need to setup 3 things:
+
+ 1. The HTML file
+ 2. The CSS file
+ 3. The JavaScript file
+
+but as we are making a music app and we want to make it beautiful, we will need some folders: 🤔
+* Images
+* Music
+
+At last be ready with your favorite songs and their album cover images.   :wink:
+
+**Initial Setup**
+Let's start by setting up our IDE (Integrated Development Environment) in Rept.it, a free, online code editor. 
+
+To start your coding right away navigate to [repl.it/languages/html](https://repl.it/languages/html) and you will have your whole setup. 
+
+You would see that there are already three files named HTML, CSS and JavaScript.  
+
+Now create the required folders of ***images*** and ***music***.
+
+![folder-creation](https://cloud-izwbr9tpa.vercel.app/0ma_1.gif)
+
+
+The set up is almost done, their one last thing we have to add that is a small library of icons which we will be needing in our app button. Do not worry, I got you covered here too! Just click the link [here](https://cloud-gsfn58rnx.vercel.app/0icons.zip), the zip folder will download and unzip it, extract the icon folder then simple drag and drop in your project.
```

Comment:
@shibam17 Check grammar and phrasing of this line(`53`) like spelling of (`there/their`) etc.

Reply:
Thanks @HariOm987 , do let me know if their is anything else! :)